README.md

```diff
@@ -1,9 +1,5 @@
 # Direct State Transfer - Go implementation (DST-GO)
 
-| Develop | Master |
-| :----: | :-----: |
-| [![CircleCI](https://circleci.com/gh/direct-state-transfer/dst-go/tree/develop.svg?style=shield)](https://circleci.com/gh/direct-state-transfer/dst-go/tree/develop) | [![CircleCI](https://circleci.com/gh/direct-state-transfer/dst-go/tree/master.svg?style=shield)](https://circleci.com/gh/direct-state-transfer/dst-go/tree/master) |
-
```

Comment:
Why is the CircleCi badge removed? It has nothing to do with the removal from golangci

Reply:
I mistakenly removed it. Have restored the badges in this commit - https://github.com/direct-state-transfer/dst-go/pull/31/commits/ef67f5e19e273644633929effc67dfdb0f93b27d.